Updated Cat 62 from 1.16 to 1.17

railsGod opened the pull request:
according to https://www.eurocontrol.int/sites/default/files/content/documents/nm/asterix/20141201-asterix-sdps-cat062-part9-v1.17.pdf

ifsnop commented:
Maybe we can leave the old version around? It can still be in use by lots of systems.

railsGod commented:
I deleted the v1.16 file in asterix/config because i found only one XML file per Cat there. (i guess the most recent one ...)

I now added the v1.17 file to install/config, what i missed initially -- v1.16 is still present there.

If you compare v1.16 (07/2012) and v1.17 (12/2014), it should be backwards compatible. Two spare bits that are zeros in v1.16 messages now mean "Code validated" and that the "Code garbled" bit is not set. Whoever is actively interpreting the value of these bits, is already using v1.17.

Thanks for the code btw.
